verification/curator-service/ui/cypress/integration/components/BulkCaseForm.spec.ts

```diff
@@ -33,6 +33,7 @@ describe('Bulk upload form', function () {
         cy.wait('@upsertCase');
         cy.wait('@upsertCase');
         cy.contains('Success! Created 2 new rows.');
+        cy.get('[data-testid="progress"]').should('not.be.visible');
```

Comment:
Could you also check that it is visible at some point?

Reply:
Not in this test, since we're making real remote calls. But I'll add a case to the unit test for this!